app/models/moderator.rb

```diff
@@ -0,0 +1,19 @@
+class Moderator
```

Comment:
Not blocking: I'm going back and forth on this, but my gut says that `community.appoint_moderator!(person)` and `community.dismiss_moderator!(person)` and `person.moderator_of?(community)` are likely better methods to describe these interactions.

Which isn't to say functions like this aren't useful, but I'm not sure we want to sprout a new class just yet?


I guess my brain is really struggling to rectify the dissonance where `Moderator` is being used to mean both `Moderator as a kind of person` and `Moderator` as a kind of relationship between a person and a community; and I was hoping to avoid conflating those things by making a Moderator model.